server/src/test/java/org/elasticsearch/rest/action/search/RestSearchActionTests.java

```diff
@@ -43,37 +39,17 @@ public void setUpAction() {
         verifyingClient.setExecuteLocallyVerifier((actionType, request) -> mock(SearchResponse.class));
     }
 
-    public void testTypeInPath() {
-        RestRequest request = new FakeRestRequest.Builder(xContentRegistry()).withHeaders(
-            Map.of("Content-Type", contentTypeHeader, "Accept", contentTypeHeader)
-        ).withMethod(RestRequest.Method.GET).withPath("/some_index/some_type/_search").build();
-
-        dispatchRequest(request);
-        assertCriticalWarnings(RestSearchAction.TYPES_DEPRECATION_MESSAGE);
-    }
-
-    public void testTypeParameter() {
-        Map<String, String> params = new HashMap<>();
-        params.put("type", "some_type");
-
-        RestRequest request = new FakeRestRequest.Builder(xContentRegistry()).withHeaders(
-            Map.of("Content-Type", contentTypeHeader, "Accept", contentTypeHeader)
-        ).withMethod(RestRequest.Method.GET).withPath("/some_index/_search").withParams(params).build();
-
-        dispatchRequest(request);
-        assertCriticalWarnings(RestSearchAction.TYPES_DEPRECATION_MESSAGE);
-    }
-
     /**
      * The "enable_fields_emulation" flag on search requests is a no-op but should not raise an error
      */
     public void testEnableFieldsEmulationNoErrors() throws Exception {
         Map<String, String> params = new HashMap<>();
         params.put("enable_fields_emulation", "true");
 
-        RestRequest request = new FakeRestRequest.Builder(xContentRegistry()).withHeaders(
-            Map.of("Content-Type", contentTypeHeader, "Accept", contentTypeHeader)
-        ).withMethod(RestRequest.Method.GET).withPath("/some_index/_search").withParams(params).build();
+        RestRequest request = new FakeRestRequest.Builder(xContentRegistry()).withMethod(RestRequest.Method.GET)
```

Comment:
these tests look like the explicit headers was just a copy/paste and not relevant to the thing it is actually testing.